Resources/Prototypes/secret_weights.yml

```diff
@@ -3,4 +3,5 @@
   weights:
     Extended: 0.25
     Nukeops: 0.25
-    Traitor: 0.75
\ No newline at end of file
+    Traitor: 0.75
+    Zombie: 0.10
```

Comment:
Should probably be about 0.05, to drop it's chances of occuring to about once-twice a day.